fuzz/run.sh

```diff
@@ -2,7 +2,7 @@
 
 handle_fail()
 {
-  let "fails++"; mkdir "$fails" ; cat out ; cat out >> fails ; mv out "$fails" ;
+  let "fails++"; mkdir "fail-$fails" ; cat out ; echo "Failed instance number $fails" >> fails ; cat out >> fails ; mv out "fail-$fails" ;
```

Comment:
Why not ...`number ${fails}"`...`mv out "fail-${fails}"`?
https://stackoverflow.com/questions/8748831/when-do-we-need-curly-braces-around-shell-variables

Reply:
in this case it doesnt matter, but I added the brackets so no mistakes will happen in future because of this

Reply:
Yes, in this case it's not error, but more about consistency and code style. I'm surprised shellcheck did not care.